mnist_image_block.py

```diff
@@ -18,10 +19,12 @@ class MNISTImageLoader(Block):
     dataset corresponding to `input_id`.
     """
 
-    version = VersionProperty('0.1.0')
+    version = VersionProperty('0.2.0')
     batch_size = IntProperty(title='Images per Batch', default=100)
     shuffle = BoolProperty(title='Shuffle Batch', default=True, visible=False)
-    validation_size = IntProperty(title='Validation Size', default=0)
+    validation_size = IntProperty(title='Validation Size',
```

Comment:
i guess not, to actually use the data reserved for validating the block would need another input node anyway. I was thinking that since we're using a non-default validation size (0 instead of 5k) that it should be user-configurable, but since the block doesn't provide access to the validation samples anyway....